docs/aqua-book/changelog.md

```diff
@@ -7,6 +7,20 @@ Aqua compiler's versioning scheme is the following: `0.BREAKING.ENHANCING.RELEAS
 * `ENHANCING` part is incremented for every syntax addition
 * `RELEASE` is the release number, shows internal compiler changes, bugfixes that keep the language untouched
 
+## [0.13.0](https://github.com/fluencelabs/aqua/releases/tag/aqua-v0.13.0) - November 22, 2023
+
+* BREAKING CHANGE: **compiler:** Make `nil` of option of bottom type ([LNG-279](https://linear.app/fluence/issue/LNG-279)) ([#968](https://github.com/fluencelabs/aqua/issues/968))
+* BREAKING CHANGE: **compiler:** Prohibit mutating options ([LNG-277](https://linear.app/fluence/issue/LNG-277)) ([#960](https://github.com/fluencelabs/aqua/issues/960))
+* **api:** Use `js.UndefOr` for `defaultServiceId` ([#980](https://github.com/fluencelabs/aqua/issues/980))
+* **compiler:** Allow omitting field name in struct creation ([LNG-261](https://linear.app/fluence/issue/LNG-261)) ([#943](https://github.com/fluencelabs/aqua/issues/943))
+* **compiler:** Always generate `last` argument of `fold` ([LNG-265](https://linear.app/fluence/issue/LNG-265)) ([#947](https://github.com/fluencelabs/aqua/issues/947))
+* **compiler:** Generate empty calls to `responseHandlerSrv` ([LNG-286](https://linear.app/fluence/issue/LNG-286)) ([#979](https://github.com/fluencelabs/aqua/issues/979))
+* **compiler:** Fix generation of wrong stream name in AIR ([LNG-276](https://linear.app/fluence/issue/LNG-276)) ([#958](https://github.com/fluencelabs/aqua/issues/958))
+* **compiler:** Fix incorrect focus on error ([LNG-274](https://linear.app/fluence/issue/LNG-274)) ([#959](https://github.com/fluencelabs/aqua/issues/959))
+* **compiler:** Fix multiple closures in one function ([LNG-262](https://linear.app/fluence/issue/LNG-262)) ([#941](https://github.com/fluencelabs/aqua/issues/941))
+* **compiler:** Fix comparing an alias and a named type with the same name ([LNG-231](https://linear.app/fluence/issue/LNG-231)) ([#946](https://github.com/fluencelabs/aqua/issues/946))
+* **compiler:** Fix that unknown service method call is ignored ([LNG-273](https://linear.app/fluence/issue/LNG-273)) ([#957](https://github.com/fluencelabs/aqua/issues/957))
```

Comment:
What was changed?

Reply:
Rephrased descriptions so that they explain what is their effect